ci-operator/step-registry/ipi/aws/ovn/hybrid-with-vxlan/ipi-aws-ovn-hybrid-with-vxlan-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: ipi-aws-ovn-hybrid-with-vxlan
```

Comment:
Here too I would name it, `ipi-aws-ovn-hybrid-custom-vxlan-port` and rename everything else associated like the sub-directories in a similar fashion.